### PR TITLE
Heaven white (black on white) theme

### LINKED ISSUE #769: Add contrast color theme - black text on white background
### Prerequisites

- [X] I'm using latest version: git latest
- [X] I've checked [Known issues](https://github.com/hovancik/stretchly#known-issues)
- [X] I've checked [Open issues](https://github.com/hovancik/stretchly/issues)

### Description

Add contrast color theme - black text on white background - white background would be clearly visible also for E-ink monitors like Dasung.

Operating system: all

### PR DESCRIPTION
**Pull request by mwoz123**
Screenshot from 2020-11-24 21-27-46

Screenshot from 2020-11-25 19-45-04

Issue: closes #769

### Requirements

* Filling out the template is required. Any pull request that does not include enough information to be reviewed in a timely manner may be closed at the maintainers' discretion.

- [x]  issue was opened to discuss proposed changes before starting implementation. It is important do discuss changes before implementing them (Why should we add it? How should it work? How should it look? Where will it be? ...).
- [x]  during development, `node` version specified in `package.json` was used (ie using [nvm](https://github.com/creationix/nvm)).
- [x]  package versions and package-lock.json were not changed (`npm install --no-save`).
- [x]  app version number was not changed.
- [ ]  all new code has tests to ensure against regressions.
- [x] `npm run lint` reports no offenses (on new code).
- [x] `npm run test` is error-free.
- [x]  README and CHANGELOG were updated accordingly.
- [ ]  after PR is approved, all commits in it are [squashed](https://gitbetter.substack.com/p/how-to-squash-git-commits)

### Description of the Change

Add contrast color theme - black text on white background - white background would be clearly visible also for E-ink monitors like Dasung.

### Verification Process

Manul tests

### CODE REVIEW

**Comment by mwoz123** [comment]
@hovancik - PR ready to be reviewed. Adapted to your comments on #776.

File: app/locales/fr.json [hunk 1]
```diff
@@ -86,7 +86,8 @@
             "menubarIcon": "Icône de la bar de menu :",
             "colour": "Couleur",
             "monochrome": "Monochrome",
-            "invertedMonochrome": "Monochrome inversé"
+            "invertedMonochrome": "Monochrome inversé",
+      "snowWhite": "Snow white"
```
Comment (hovancik) [inline app/locales/fr.json:90]: now that we use Weblate for translations, it's actually better to leave untranslated strings out. So where strings are not translated yet, please remove them.

File: app/css/preferences.css [hunk 2]
```diff
@@ -209,13 +213,21 @@ body > .hidden,
...
 .appearance > #greenClouds:checked + label:before,
 .appearance > #autumnBeBlessed:checked + label:before,
 .appearance > #graphiteCrystal:checked + label:before,
 .appearance > #coffeeKisses:checked + label:before,
 .appearance > #morningSwim:checked + label:before {
   color: #FFFFFF;
 }
+.appearance > #snowWhite:checked + label:before {
+  color: #000000;
+}
```
Comment (hovancik) [inline app/css/preferences.css:230]: You'll need to edit grid, notice the two lines in Theme preferences  have bad location comparing to current version.

**Review by hovancik — CHANGES_REQUESTED** [review]
2 small things :) Otherwise it looks good to me

File: app/css/preferences.css (hunk 2 above)
Comment (mwoz123) [reply app/css/preferences.css:230]: didn't spot it. thanks

File: app/locales/fr.json (hunk 1 above)
Comment (mwoz123) [reply app/locales/fr.json]: done

**Comment by mwoz123** [comment]
done. Please re-review

File: app/locales/de.json [hunk 3]
```diff
@@ -86,7 +86,7 @@
             "menubarIcon": "Symbol in Menüleiste:",
             "colour": "Farbig",
             "monochrome": "Einfarbig",
-            "invertedMonochrome": "Einfarbig Invertiert"
+            "invertedMonochrome": "Einfarbig Invertiert",
```
Comment (hovancik) [inline app/locales/de.json:89]: one small thing :) you have extra `,` left after removing the unwanted strings.

**Review by hovancik — CHANGES_REQUESTED** [review]

File: app/locales/de.json (hunk 3 above)
Comment (mwoz123) [reply app/locales/de.json]: right I was too fast.

**Comment by mwoz123** [comment]
fixed please recheck

**Review by hovancik — APPROVED** [review]

**Comment by mwoz123** [comment]
thanks

**Pull request merged**